magprime/models.py

```diff
@@ -2,18 +2,6 @@
 
 
 @Session.model_mixin
-class Group:
-    @presave_adjustment
-    def bucket_pricing_workaround(self):
```

Comment:
Getting rid of this is our main goal, as it makes group functionality unpredictable for admins.